tutorials/ngsi_v2/e1_virtual_weatherstation/e1_virtual_weatherstation_solution.py

```diff
@@ -31,13 +31,12 @@
 # import simulation model
 from tutorials.ngsi_v2.simulation_model import SimulationModel
 
```

Comment:
It is a general remark. Could you make the same modification also for the exercise scripts? Like for `e1_virtual_weatherstation.py`. I showed an example for `e5`

Reply:
My idea was to get the solutions to work first and then of course reformat the exercise scripts according to the solutions. I will do it